src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -256,6 +256,11 @@ export default (accessToken, userID, opts) => {
       {},
       { method: "PUT" }
     ),
+    updateViewingRoomLoader: gravityLoader(
+      (id) => `viewing_room/${id}`,
+      {},
+      { method: "PUT" }
+    ),
```

Comment:
Might need a rebase with origin/master to not duplicate the changes from https://github.com/artsy/metaphysics/pull/6267 in the diff.